docs/pages/access-controls/access-request-plugins/ssh-approval-discord.mdx

```diff
@@ -35,6 +35,8 @@ in your Teleport cluster.
 
 (!docs/pages/includes/plugins/rbac.mdx!)
 
+(!/docs/pages/includes/plugins/rbac-impersonate.mdx!)
```

Comment:
Both the `rbac.mdx` and `rbac-impersonate.mdx` partials have the same impersonation instructions, it looks like, so there's some repetition in this guide. I'd check this and the other Access Request guides edited by this PR to make the guides flow as expected.